src/CustomMarshalers/src/System/Runtime/InteropServices/CustomMarshalers/EnumerableToDispatchMarshaler.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+
+namespace System.Runtime.InteropServices.CustomMarshalers
+{
+    public class EnumerableToDispatchMarshaler : ICustomMarshaler
```

Comment:
Also, do these types/APIs need to go through api-review or can that be skipped since it is a direct port?

Reply:
For the types that are exposed it's the same API surface. There are a few types that are present in .NET Framework that are not present here since they aren't applicable to .NET Core (either the types don't exist or the functionality would need to be stubbed out). I don't think we would need an API review.